docs/deployment/configuration.mdx

```diff
@@ -135,6 +135,7 @@ OpenAPI configuration is used for integrating with OpenAI services. These settin
 | Env var | Purpose | Required | Default Value | Valid options |
 |:-------------------:|:-------:|:----------:|:-------------:|:-------------:|
 | **OPENAI_API_KEY** | API key for OpenAI services | No | None | Valid OpenAI API key |
+| **OPENAI_BASE_URL** | URL for OpenAI API. It's need for use yourself LOCAL LLM instead OpenAi LLM. | No | None | Valid OpenAI API url |
```

Comment:
Please provide examples here, I assume default value is not None.

Reply:
Found in the lib code, it's `https://api.openai.com/v1`

Reply:
@Matvey-Kuk can this be merged in?